app/components/goal-block/component.js

```diff
@@ -6,19 +6,23 @@ const {
   observer
 } = Ember;
 
+const SingleOptionWidth = 64;
+
```

Comment:
I believe the convention is to use UPPER_CASE for constants defined in a module. Capitalised camel-case is used for 'class' names.